invenio/base/templates/admin_header_base.html

```diff
@@ -0,0 +1,89 @@
+{#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+<header class="main-header">
+{% block headerbox %}      
+  <a href="{{ url_for('collections.index') }}" class="navbar-brand logo">
+    <img src="{{ url_for('static', filename='img/logo_white.svg') }}" alt="{{ config.CFG_SITE_NAME_INTL[g.ln] }}" />
+  </a>
+  <!-- Header Navbar: style can be found in header.less -->
```

Comment:
If you don't mind, I just remove this comment. It remained from the template and it's not really necessary.